app.json

```diff
@@ -65,5 +68,8 @@
     {
       "url": "https://github.com/heroku/heroku-buildpack-cli"
     }
+  ],
+  "addons": [
+    "mongolab"
```

Comment:
I think thIs addon will add `MONGO_URI` variable directly to the environment. I'm not sure
- if it does there is no point of asking separately in deployment form
- if not what is the use of this addon
Please clarify my doubt